tests/files/calc-invalid.css

```diff
@@ -0,0 +1,22 @@
+div{
+  height: calc (25% - 1em);
+}
+
+div{
+  height: calc
+  (25% - 1em);
+}
+
+div{
+  height: calc
```

Comment:
I don’t think we should disallow `calc` used as a non-function keyword (I realize your change didn’t break this and simply improved the error reporting) but to my knowledge the following is valid:

```css
#outer {
  grid-template-areas: "top calc" "bottom calc";
}
#inner {
  grid-area: calc;
}
```

Reply:
Or maybe I misunderstood… is the error caused by the missing semicolon at the end?

Reply:
It appears that
```CSS
#inner {
  grid-area: url;
}
```
is as valid. I pushed an update that handles this and as a side effect it also fixes #174